indico/htdocs/js/indico/modules/registration/form/tpls/fields/dialogs/size.tpl.html

```diff
@@ -9,3 +9,32 @@
         max="60">
     <i class="info-helper" title="{{ 'Must be between 5 and 60' | i18n }}"></i>
 </div>
+<ng-if="field.inputType == 'text'>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Max length in char' | i18n }}
+        </label>
+        <input type="number" name="maxlength"
+            ng-model="formData.maxLength"
+            >
+        <i class="info-helper" title="{{ 'Maximimum  allowed length' | i18n }}"></i>
+    </div>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Min length in char' | i18n }}
+        </label>
+        <input type="number" name="minlength"
+            ng-model="formData.minLength"
+            >
+        <i class="info-helper" title="{{ 'Minimimum  allowed length' | i18n }}"></i>
+    </div>
+    <div class="regFormEditLine">
+        <label class="regFormDialogCaption">
+            {{ 'Validation pattern' | i18n }}
+        </label>
+        <input type="text" name="validpattern"
+            ng-model="formData.validPattern"
+            >
```

Comment:
should go in the previous line (also two more times above)

Reply:
done